locales/en-US/server.ftl

```diff
@@ -0,0 +1,161 @@
+shotPageAlertErrorDeletingShot = Error deleting shot: {status} {statusText}
+shotPageAlertErrorUpdatingTitle = Error saving title: {status} {statusText}
+
+shotPageShareButton
+  .title = Share
+shotPageCopy = Copy
+shotPageCopied = Copied
+shotPageShareFacebook
+  .title = Share to Facebook wall or message
+// TODO: awkward phrase, maybe 'Share on Twitter'?
+shotPageShareTwitter
+  .title = Share to a tweet
+shotPageSharePinterest
+  .title = Share to Pinterest
+shotPageShareEmail
+  .title = Create email with link
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = This shot is only visible to you until you share the link.
+shotPageCopyImageText
+  .label = Copy Image Text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {timediff} is a placeholder for a future time like 'tomorrow' or 'next week'
+// TODO: "in {timediff}" probably won't work across languages. maybe an absolute date is best.
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted in {timediff}.
+// Note: {timediff} is a placeholder for a time duration phrase like '30 days'
+// TODO: again, "for {timediff}" may not work well.
+shotPageRestoreButton = restore for {timediff}
+shotPageExpiredMessage = This shot has expired.
+shotPageExpiredMessageDetails = Here is the page it was originally created from: {page}
+shotPageDeleteButton
+  .title = Delete this shot permanently
+shotPageAbuseButton
+  .title = Report this shot for abuse, spam, or other problems
+shotPageMyShots = My Shots
+shotPageDownloadShot
+  .title = Download the shot image
+shotPageDownload = Download
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageScreenshotsDescription  = Firefox Screenshots made simple. Take, save and share screenshots without leaving Firefox.
+shotPageUpsellFirefox = Get Firefox now
+shotPageDoesNotExpire = does not expire
+
+// shotindex page
+
+// {status} is a placeholder for an HTTP status code, like '500'.
+// {statusText} is a text description of the status code, like 'Internal server error'.
+shotIndexPageErrorDeletingShot = Error deleting shot: {status} {statusText}
+// {searchTerm} is a placeholder for text the user typed into the search box
+shotIndexPageSearchResultsTitle = My Shots: search for {searchTerm}
+// {error} is a placeholder for a non-translated error message that could be shared
+// with developers when debugging an error.
+shotIndexPageErrorRendering = Error rendering page: {error}
+shotIndexPageSearchPlaceholder
+  .placeholder = search my shots
+shotIndexPageSearchButton
+  .title = search
+shotIndexPageNoShotsMessage = No saved shots.
+shotIndexPageNoShotsInvitation = Go on, create some.
+// TODO should we include the ellipsis as three periods, or leave a placeholder for an html entity?
+shotIndexPageLookingForShots = Looking for your shots...
```

Comment:
The comment was just a reminder to myself to ask about the unicode ellipsis, which you already mentioned in another comment :+1: